deb/build/debian/jenkins.init

```diff
@@ -43,6 +46,26 @@ if [ "$RUN_STANDALONE" = "false" ]; then
     exit 1
 fi
 
+# Make sure there exists a java executable, it may not be allways the case
+if [ -z "$JAVA" ]; then
```

Comment:
Won't Jenkins just gracefully use `$USER/.jenkins` if `$JENKINS_HOME` is not set?